pkg/services/resource.go

```diff
@@ -91,9 +96,9 @@ func (s *sqlResourceService) Create(ctx context.Context, resource *api.Resource)
 	return resource, nil
 }
 
-func (s *sqlResourceService) Replace(ctx context.Context, resource *api.Resource) (*api.Resource, *errors.ServiceError) {
+func (s *sqlResourceService) Update(ctx context.Context, resource *api.Resource) (*api.Resource, *errors.ServiceError) {
 	if !DisableAdvisoryLock {
```

Comment:
i hadn't seen this flag.  when would we disable the lock?

i tend to think that composition and DI are better than flags.

Reply:
Looks like the flag will be removed in https://github.com/openshift-online/maestro/pull/7
I will remove this, too.